app/models/room.rb

```diff
@@ -1,3 +1,5 @@
 class Room < ApplicationRecord
    belongs_to :hotel
+   has_many :room_guests
+   has_many :guests, through: :room_guests
 end
```

Comment:
All the associations are set up correctly. 

I want to take the opportunity to recommend that the `guests.count` that you're currently doing in the view actually be made into a model method in the `Room` model. The way it's set up, you're actually performing ActiveRecord in your view which breaks MVC convention.